### ISSUE TITLE
Error with audit hook for kmeans model due to learning_rate

### ISSUE THREAD

**Issue by rbjerrum** [kristeligt-dagblad/dbt_ml#5]
The audit hook attempts to get the learning rate for the kmeans model. This is not a supported BQ operation.
